exporter/observiqexporter/converter.go

```diff
@@ -14,23 +14,36 @@
 
 package observiqexporter
 
-import "go.opentelemetry.io/collector/model/pdata"
+import (
+	"crypto/md5" //nolint:gosec // Not used for crypto, just for generating ID based on contents of log
```

Comment:
Is it required by your backend, or can it be anything? There are several hashing functions that could be appropriate here.

Reply:
MD5 isn't really a hard requirement, any hash function should work. I used MD5 here because it's what we use in our own agent, currently. If you have a suggestion for a hash function you think would fit better (maybe one of those BLAKE2 hash functions?), I'd be happy to hear it!

Reply:
How about fnv? https://golang.org/pkg/hash/fnv/

Reply:
Good suggestion. Seems like it'll work just fine for our purpose, I've switched over to it in favor of md5 here.